### PR TITLE
Add 4CCs

### PR DESCRIPTION
**Pull request by justtryingthingsout**
Most of the added codenames are taken from bputil.

### CODE REVIEW

File: _data/4cc.yaml [hunk 1]
```diff
@@ -40,19 +40,19 @@
...
+  description: Chip Security Mode
   usage:
   - APTicket
 - code: ECID
   description: Exclusive Chip ID
   usage:
   - APTicket
 - code: EPRO
-  description:
+  description: Effective Production Status
```
Comment (rickmark) [inline _data/4cc.yaml:51]: Is this production as in demotion?

File: _data/4cc.yaml [hunk 2]
```diff
@@ -188,7 +189,7 @@
   usage:
   - Boot Policy
 - code: lpnh
-  description: LocalPolicy Nonce Hash
+  description: Local Policy Nonce Hash
```
Comment (rickmark) [inline _data/4cc.yaml:192]: LocalPolicy is stylized that way in usage

**Review by rickmark — APPROVED** [review]
Few bits but improves quality so merging as is

File: _data/4cc.yaml (hunk 1 above)
Comment (justtryingthingsout) [reply _data/4cc.yaml:51]: yes, like CPRO

File: _data/4cc.yaml (hunk 2 above)
Comment (justtryingthingsout) [reply _data/4cc.yaml:192]: apparently not for bputil

File: _data/4cc.yaml (hunk 1 above)
Comment (rickmark) [reply _data/4cc.yaml:51]: If I understand CPRO would be 1 (true) and EPRO would be 0 (false) if it’s a prod device demoted?

File: _data/4cc.yaml (hunk 2 above)
Comment (rickmark) [reply _data/4cc.yaml:192]: Fair - depends on context (I might make this “Local Policy `LocalPolicy`) for search?

**Pull request merged**